docs/topics/braintree.rst

```diff
@@ -261,3 +260,27 @@ Some information about the subscripton is stored in solitude.
     :query paymethod__braintree_buyer: the primary key of the braintree buyer.
     :query paymethod__braintree_buyer__buyer: the primary key of the buyer.
     :query seller_product: the primary key of the product.
+
+Webhook
+-------
+
+When Braintree completes certain actions, webhooks will be sent. For more information
```

Comment:
maybe more specific like: "When Braintree completes...they will post data to our configured webhook URL"